src/app/feature-flag/service/feature-toggles.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Location } from '@angular/common';
+import { Inject, Injectable } from '@angular/core';
+import { Headers, Http } from '@angular/http';
+import { Logger } from 'ngx-base';
+import { WIT_API_URL } from 'ngx-fabric8-wit';
+import { AuthenticationService } from 'ngx-login-client';
+import { Observable } from 'rxjs';
+//import { FABRIC8_FEATURE_TOGGLES_API_URL } from '../../../a-runtime-console/shared/feature-toggles.provider';
+
+@Injectable()
+export class FeatureTogglesService {
+  private headers = new Headers({ 'Content-Type': 'application/json' });
+  private featureTogglesUrl: string;
+
+  constructor(
+    private http: Http,
+    private logger: Logger,
+    private auth: AuthenticationService,
+    //@Inject(FABRIC8_FEATURE_TOGGLES_API_URL) apiUrl: string) {
+    @Inject(WIT_API_URL) apiUrl: string) {
+    if (this.auth.getToken() != null) {
+      this.headers.set('Authorization', 'Bearer ' + this.auth.getToken());
```

Comment:
We no longer need to provide the auth token ourselves. I believe there is a global HTTP authentication service that we should use instead?

Reply:
nope no http service to add tokens. AuthenticationService provide getToken to be used for that purpose. there is something close `Fabric8UIHttpService` but it's not adding the token.

Reply:
@corinnekrych  - that's not how it's supposed to work - the auth http service is injected at the root which provides header adjustments for all http calls to wit, sso and auth endpoints.  See line 47 in fabric8-ui-http.service.ts.  If it's not working for you, it means that the module you are in has overridden the http service incorrectly.  

You can read more in ngx-login-client http.service in the request function which is injected globally inside the platform.  We should diligently remove any code that is adding a token manually so the http requests can all be consistent instead of manually coded this way.

Reply:
@corinnekrych - The http service that is provided by ngx-login-client does add tokens for you.  We should not use auth.getToken() because the service I mentioned above injects it automatically.  We can let this in if it's not working for you, but we should add an issue to fix it as this is definitely a bug.